OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -63,6 +63,8 @@ public class ArmamentInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>
 		[Desc("Use multiple muzzle images if non-zero")]
 		public readonly int MuzzleSplitFacings = 0;
 
+		public readonly bool ShootAtClosestTargetablePosition = false;
```

Comment:
My point is that adding a new entry later would be technically wrong, because there would be no functional difference (the weapon/projectile doesn't care if the list of targetable positions was derived from a footprint or some other, custom list of positions).

Could you and @pchote live with calling it either `MultiplePositions` or `ClosestPosition` instead of `Footprint`, to account for the fact that it won't be limited to footprints?